docs/pull_request_template.md

```diff
@@ -13,3 +13,5 @@
 - https://[HEROKU-APP-ID].herokuapp.com/uk-nationals-living-eu
 
 [Other finders](https://live-stuff.herokuapp.com/finders)
+
+Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a major Rails upgrade.
```

Comment:
Maybe remove this one? We only need it to be present in one of the templates since the workflow will concatenate all the templates together before posting it as a comment on dependabot PRs

Reply:
> Maybe remove this one? We only need it to be present in one of the templates since the workflow will concatenate all the templates together before posting it as a comment on dependabot PRs

Done